src/app/services/views/MeshServiceListView.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <DataSource
+    v-slot="{ data: me }: MeSource"
+    src="/me"
+  >
+    <RouteView
+      v-if="me"
+      v-slot="{ route, t, uri }"
+      name="mesh-service-list-view"
+      :params="{
+        page: 1,
+        size: me.pageSize,
```

Comment:
```suggestion
        size: me.size,
```
Should it be `size`? I don't see `pageSize` defined anywhere.